test/unit/gateways/sage_pay_test.rb

```diff
@@ -186,6 +186,20 @@ def test_successful_store
     assert_equal '1', response.authorization
   end
 
+  def test_successful_verify
+    response = stub_comms do
+      @gateway.verify(@credit_card, @options)
+    end.respond_with(successful_purchase_response)
```

Comment:
Since `verify` is using `authorize` and `void` under the covers, I'm thinking you'll want to use those rather than `successful_purchase_response`.  Similar to this:

https://github.com/Shopify/active_merchant/blob/master/test/unit/gateways/merchant_partners_test.rb#L201-L215